vendor/github.com/ethereum/go-ethereum/p2p/discv5/net.go

```diff
@@ -40,8 +40,9 @@ var (
 )
 
 const (
-	autoRefreshInterval   = 1 * time.Hour
-	bucketRefreshInterval = 1 * time.Minute
+	autoRefreshInterval = 1 * time.Hour
+	// spams the network, we do enough lookups when topic is being search
+	bucketRefreshInterval = 120 * time.Minute
```

Comment:
Can you elaborate on where this estimation comes from? Better to put it into PR description, so it's discoverable later on.

Reply:
Best in PR description plus a short comment in code for later maintainers don't looking into the PR history.